### PR TITLE
2608: Add purpose strings for calendar permissions, use Sie for malte help form and add translations

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Add purpose strings for calendar permissions, use Sie for malte help form and add translations.

### Proposed changes

- [x] Use Sie for malte help form
- [x] Submit for translations
- [ ] Add translations
- [ ] Add purpose strings for calendar permissions to `ios/Integreat/<lang>.lproj` files

### Side effects

None.

### Resolved issues

Partly fixes: #2608.

---

### CODE REVIEW

**Comment by steffenkleinle** [comment]
Adding translations and purpose strings to ios lang files will be done in a separate PR.

File: translations/translations.json [hunk 1]
```diff
@@ -3125,7 +3125,6 @@
       "negativeRating": "ይህ ገጽ ጠቃሚ አይደለም",
       "positiveRating": "ይህ ገጽ ጠቃሚ ነው",
       "contactMailAddress": "ለጥያቄዎች ኢሜይል",
-      "optionalInfo": "አማራጭ /ሊተው የሚችል/",
```
Comment (LeandraH) [inline translations/translations.json:3128]: Why did you delete this one?

**Review by LeandraH — APPROVED** [review]
Looks good, not tested

File: translations/translations.json (hunk 1 above)
Comment (steffenkleinle) [reply translations/translations.json:3128]: Because it was just a leftover from a previous merge, it was moved to `optional` in common.

**Review by ztefanie — APPROVED** [review]
